src/emails/templates/EmailFooter.tsx

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import React from "react";
```

Comment:
Next.js auto-imports React in JSX files, but esbuild doesn't (at least no by default, maybe there's an option somewhere).